src/components/cl/hier/alltoallv/alltoallv.c

```diff
@@ -122,6 +125,7 @@ UCC_CL_HIER_PROFILE_FUNC(ucc_status_t, ucc_cl_hier_alltoallv_init,
 {
     ucc_cl_hier_team_t     *cl_team = ucc_derived_of(team, ucc_cl_hier_team_t);
     ucc_cl_hier_lib_t      *cl_lib  = UCC_CL_HIER_TEAM_LIB(cl_team);
+    int                     full_only = 0;
```

Comment:
minor: with the following suggestions the variable `full_only` is not used anymore

Reply:
still needed for frees